### PR TITLE
Switch from Maven to Gradle.

### PR DESCRIPTION
**Pull request by dumptruckman**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    }
+
+    api('me.main__.util:SerializationConfig:1.7') {
+        exclude group: 'org.bukkit', module: 'bukkit'
+    }
+    api('com.pneumaticraft.commandhandler:CommandHandler:11') {
+        exclude group: 'org.bukkit', module: 'bukkit'
+        exclude group: 'junit', module: 'junit'
+    }
+    api 'com.dumptruckman.minecraft:buscript:2.0-SNAPSHOT'
```
Comment (zax71) [inline build.gradle:60]: May want to remove this as Buscript has been removed now in the MV5 branch

**Review by zax71 — COMMENTED** [review]
Just one thing

File: build.gradle (hunk 1 above)
Comment (benwoo1110) [reply build.gradle:60]: This is targeting main, so it should be kept. I will change when I merge MV5

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -13,10 +13,17 @@ jobs:
...
-    - name: Build with Maven
-      run: mvn -B package --file pom.xml
+    - name: Build with Gradle
+      run: gradlew build
+
+    - name: Upload Artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: Multiverse-Core-SNAPSHOT
+        path: build/libs/Multiverse-Core-4.3.2-SNAPSHOT.jar
```
Comment (benwoo1110) [inline .github/workflows/build.yml:30]: The version `4.3.2-SNAPSHOT` probably shouldn't be hard coded

Comment (zax71) [reply .github/workflows/build.yml:30]: oops, good spot

**Pull request merged**